.github/workflows/deploy_latest_release.yml

```diff
@@ -45,13 +45,15 @@ jobs:
             project: pwa-kit
             mobify_user: MOBIFY_STG_CLIENT_USER
             mobify_api_key: MOBIFY_STG_CLIENT_API_KEY
-            flags: --cloud-origin https://cloud-testing.mobify-staging.com -c ~/.mobify --wait
+            cloud_origin: https://cloud-testing.mobify-staging.com
```

Comment:
Bonus: refactor the flags for this workflow too.

Reply:
We no longer need to pass in the `~/.mobify` path for the credentials file, because it's now the default value defined in the action.